src/Microsoft.ML.StaticPipe/ModelOperationsCatalog.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.IO;
+
+namespace Microsoft.ML.StaticPipe
+{
+    public static class ModelOperationsCatalog
```

Comment:
>ModelOperationsCatalog [](start = 24, length = 22)

It may be preferable if the static class containing the extension methods for this did not have the same name as the type they're operating on, that is `ModelOperationsCatalogExtensions` or suchlike may be preferable.

Reply:
fixed